tests/Commands/AzureISV/Datadog/MonitoredResourcesListCommandTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
Looks good.  Let's get some live tests added.  @hallipr has a guide on how to do that.

Reply:
Here's a PR with them: 

The test: https://github.com/Azure/azure-mcp/pull/144/files#diff-e57d72d894c52aa656d1560a288353cab6ec627aa9e771cbaf0590871c62c59e

Adding infra to create resource: https://github.com/Azure/azure-mcp/pull/144/files#diff-2dfdd33f7deeb26d50a329a65a1e5009ab6955bbfec3575dd4323fe1fda90ae2

Instructions on how to run them: https://github.com/Azure/azure-mcp/blob/main/CONTRIBUTING.md#live-tests